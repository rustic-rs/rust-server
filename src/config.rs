pub mod auth_config;
pub mod auth_file;
pub mod server_config;
