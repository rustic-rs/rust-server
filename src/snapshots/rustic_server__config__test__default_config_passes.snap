---
source: src/config.rs
expression: config
---
RusticServerConfig(
  server: ConnectionSettings(
    listen: Some("127.0.0.1:8000"),
  ),
  storage: StorageSettings(
    r#data-dir: Some("C:\\Users\\dailyuse\\AppData\\Local\\Temp\\rustic"),
  ),
  auth: HtpasswdSettings(
    r#disable-auth: false,
  ),
  acl: AclSettings(
    r#disable-acl: false,
    r#append-only: true,
  ),
  tls: TlsSettings(
    r#disable-tls: true,
    r#tls-key: None,
    r#tls-cert: None,
  ),
  log: LogSettings(),
)
